src/components/dropdown-search/dropdown-search.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./dropdown-search.scss";
+import ArrowDown from "../../images/icons/arrow-down.svg";
+import { SearchInput } from "../search-input/search-input";
+
+export const DropdownSearch = ({
+  title,
+  options,
+  onSelect,
+  isAlwaysOpen,
+  showSearchInput = true,
```

Comment:
It gets the default value from `defaultProps`, please remove it from here

Reply:
Done.